com.unity.render-pipelines.high-definition/Runtime/Lighting/GlobalIllumination.cs

```diff
@@ -83,7 +83,7 @@ public bool fullResolutionSS
             set { m_FullResolutionSS.value = value; }
         }
         [SerializeField]
-        public BoolParameter m_FullResolutionSS = new BoolParameter(true);
+        private BoolParameter m_FullResolutionSS = new BoolParameter(true);
```

Comment:
if you do this, it mean we can't control the value frpom C#, is it what we want?

Reply:
Check the block just above that declares public bool fullResolutionSS